src/components/App/App.component.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
```

Comment:
Try to search for a solution where `test` and `expect` aren't marked as undefined by VSCode without using an ESLint comment.

Reply:
These comments were fixed on[ PR3](https://github.com/laulujan/react-apprenticeship-capstone1/pull/3), I had missing `jest: true,node: true,` on eslintrc.js file and that's why I was getting those linting errors.